js/wizard/wizard.js

```diff
@@ -1,21 +1,32 @@
 import { chooseFighterStep } from './steps/choose-fighter-step/choose-fighter-step.js';
 import { bookingDetailsStep } from './steps/booking-details-step/booking-details-step.js';
 import { bookingConfirmationStep } from './steps/booking-confirmation-step/booking-confirmation-step.js';
+import { buttonMaker } from '../components/buttonMaker.js';
 
-export const initWizard = () => {
+export const initWizard = (index) => {
+   let button = buttonMaker('choose');
```

Comment:
**nitpick:** Im not a fan of your `buttonMaker` function, as a developer who does not know your implementation it's hard to predict what will happen if I pass string there. I suggest to parametrize this `buttonMaker` by passing `props` object, or `params` - where you will keep all needed stuff to create your button. Example, your params object could look like this:

```js
let button = buttonMaker({
  onClick: () => {},
  imageSrc: "/some/path",
  style: {
    background: "red"
  },
})
```